packages/manager/apps/adp/index.js

```diff
@@ -1,9 +1,12 @@
 import 'script-loader!jquery'; // eslint-disable-line
 import 'script-loader!lodash'; // eslint-disable-line
+import 'ovh-ui-kit/dist/oui.css';
 import '@ovh-ux/manager-adp';
+import ovhManagerCore from '@ovh-ux/manager-core';
 
 import './adp.less';
 
 import angular from 'angular';
 
-angular.module('adpApp', ['ovhManagerAdp']);
+angular.module('adpApp', ['ovhManagerAdp', ovhManagerCore])
```

Comment:
Must be defined as a module dependency of `packages/manager/modules/adp/src/index.js` instead.

Note that `ovhManagerCore` is already declared as a module dependency of the root application available in that following path `packages/manager/apps/public-cloud/`.